manual/src/ornate/cookbook.md

```diff
@@ -280,3 +280,14 @@ reload on every change:
 ~~~
 webpackDevServerExtraArgs := Seq("--inline")
 ~~~
+
+## How to pass extra parameters to webpack
+
+You can add extra params to be passed to webpack, e.g. to increase debugging
+
+~~~
+webpackExtraArgs := Seq("--profile", "--progress", "true")
+~~~
+
+**Note** Params are passed verbatim, they are not sanitized and could create errors when passed to webpack.
+In particular avoid setting the "--config" param.
```

Comment:
I would rather explain at the beginning what the plugin does when it invokes webpack (= it sets the `--config` option to pick the right configuration file), but it is possible to set extra arguments with `webpackExtraArgs` (to set something *else* than `--config`).